ngraph/core/reference/include/ngraph/runtime/reference/divide.hpp

```diff
@@ -110,8 +112,20 @@ namespace ngraph
                         return x / y;
                     }
                 };
-                autobroadcast_binop(
-                    arg0, arg1, out, arg0_shape, arg1_shape, broadcast_spec, functor);
+
+                if (arg0_shape == arg1_shape)
+                {
+                    const auto elems = shape_size(arg0_shape);
+                    perf::Timer t{"Elementwise division with " + std::to_string(elems) +
```

Comment:
Why did you add a timer to divide but not to add operator?

Reply:
I did it to demonstrate how I measured the time of execution. I've removed the measurements before submitting the PR. In the end all of the timer utilities will be removed and the intention is to just add the executor to the code base.